runtime/compiler/runtime/J9Profiler.cpp

```diff
@@ -1643,7 +1643,9 @@ TR_BlockFrequencyInfo::TR_BlockFrequencyInfo(TR_CallSiteInfo *callSiteInfo, int3
    _blocks(blocks),
    _frequencies(frequencies),
    _counterDerivationInfo(NULL),
-   _entryBlockNumber(-1)
+   _entryBlockNumber(-1),
+   _isQueuedForRecompilation(0),
+   _isQueuedForRecompilationSymRef(NULL)
```

Comment:
This structure is a persistent structure - the symref should not be stored here - normally we would have a findOrCreate on the SymRefTab - because the pointer will be stale once the compile terminates it certainly doesn't belong here.

Reply:
Removed SymRef : https://github.com/eclipse/openj9/pull/2346/commits/42b3027ce7646151036f0700b8d9ce9e90f40197